config/redirects

```diff
@@ -249,3 +250,7 @@ raw: docs/drivers/community-supported-drivers/ -> ${base}/
 raw:  docs/drivers/php/ -> ${base}/php-drivers/
 raw:  docs/drivers/python/ -> ${base}/python-drivers/
 raw:  docs/drivers/ruby/ -> ${base}/ruby-drivers/
+
+# Docs consolidation redirects
+
+raw: ${base}/pymongo/ -> ${pymongo-base}
```

Comment:
My understanding is that this will also redirect all subpages. Will confirm after merging and deploying.